### PR TITLE
Cart2

### PR DESCRIPTION
**Pull request by saba-mo**
### Assignee Tasks

* [x] cart component and store 
* [x] confirmation component
* [x] cart api route

### Guidelines

Outstanding issues: once associations are set in db, can work on eager loading products into cart

### CODE REVIEW

File: client/components/confirmation.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Confirmation = () => {
+  return (
+    <div>
+      <h2>Confirmation</h2>
+      {/* what do we want to show the customer on the confirmation page? */}
```
Comment (abstauff) [inline client/components/confirmation.js]: Delete this comment

File: client/components/index.js [hunk 2]
```diff
@@ -11,4 +11,5 @@ export {default as AllProducts} from './products/AllProducts';
 export {default as SingleProduct} from './products/SingleProduct';
 export {default as AllUsers} from './users/UserList';
 export {default as SingleUser} from './users/SingleUser';
-//export {default as Cart} from './cart'
+export {default as Cart} from './Cart';
```
Comment (abstauff) [inline client/components/index.js]: Add a "cart" folder with confirmation and cart inside.

File: server/api/cart.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+const router = require('express').Router();
+const Sequelize = require('sequelize');
+const Product = require('../db/models/product');
+const Cart = require('../db/models/cart');
+const Order = require('../db/models/order');
+
+// unsure if we need this route - shows everyone's carts
+router.get('/', async (req, res, next) => {
```
Comment (abstauff) [inline server/api/cart.js]: Not needed at the moment...

File: server/api/cart.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  } catch (error) {
+    next(error);
+  }
+});
+
+router.get('/:cartId', async (req, res, next) => {
+  try {
+    const thisCart = await Cart.findAll({
+      where: {
+        order_id: req.params.cartId
```
Comment (abstauff) [inline server/api/cart.js]: parseInt(req.params.cartId)

**Review by abstauff — COMMENTED** [review]
Do some stuff, push it again!

File: client/components/cart/Cart.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import {connect} from 'react-redux';
-import {fetchCart} from '../store/redux/cart';
+import {fetchCart} from '../../store/redux/cart/cart';
 
 class Cart extends React.Component {
   constructor() {
```
Comment (ahillary) [inline client/components/cart/Cart.js]: add documentation to line 22 letting the group know that the functions are working with your db where you forced in associations.

**Review by ahillary — APPROVED** [review]

**Pull request merged**